content/finding-command-help.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Find Command Help Using the Terminal
+description: >
+  Using apropos, whatis, info, man, and --help to find appropriate commands and command usage information.
+keywords:
+  - Linux
+  - Beginner
+  - Linux Terminal
+  - Command Help
+  - System76
+  - Technical Support
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: about-your-os
+tableOfContents: true
+---
+
+Even experienced Linux users frequently utilize knowledge resources to find proper syntax for Terminal commands, or to discover options for known commands. All Linux shells support common utilities that search for and explain commands for a given task. These commands may navigate the user to a text file or navigable structure of help files, while others simply print output into the current Terminal session.
+
+### Why Find Help Using the Terminal?
+
+Issues may disable user access to the desktop environment, including the web browser. In these scenarios, users can still find help that displays as output in a Terminal session. Users who still have access to a browser or second computer can find help using the [Ubuntu](https://manpages.ubuntu.com/) or [Debian](https://manpages.debian.org/) manpage websites. The commands below are entered in a Terminal session. In Pop!\_OS, Terminal sessions can be started by pressing <kbd>Super</kbd> + <kbd>t</kbd>.
+
+### --help
+
+`--help` prints out a shortened version of the man page, usually including a brief example and all options for a command. `--help` prints output to the terminal session without navigating away from the shell prompt.
+
+**Use case**: A user wants to quickly reference all options for a known command in the current Terminal session.
+
+**Usage**: Type a command + `--help`, then hit <kbd>Enter</kbd>. This example uses the `cd` command. Commands must be an exact match.
+```bash
+cd --help
+```
+![--help output](/images/finding-command-help/--help-output.png)
+
+### whatis
+
+`whatis` provides a very short description of a command's functionality.
+
+**Use case**: A user wants a very brief reminder of a known command's functionality printed in the current Terminal session.
+
+**Usage**: Type `whatis` + command, then hit <kbd>Enter</kbd>. This example uses the `mkdir` command. Commands must be an exact match.
+```bash
+whatis mkdir
+```
+![whatis output](/images/finding-command-help/what-output.png)
+
+### which
+
+`which` provides the directory location of a command.
+
+**Use case**: A user would like to know which installed package provides a given command.
+
+**Usage**: Type `which` + command(s), then hit <kbd>Enter</kbd>. This example uses the ls and dpkg commands.
+```bash
+which ls -l dpkg -S
```

Comment:
Including flags in a `which` command doesn't make sense; see my last two comments on the conversation thread and feel free to modify/remove/update however you see fit.